### PR TITLE
fix #173 cloudmergin to merginmaps.com

### LINKED ISSUE #173: Replace cloudmergin.com links with merginmaps.com
There are quite a few places pointing to old `public.cloudmergin.com` like:
- https://merginmaps.com/docs/setup/sign-up-to-mergin-maps/#from-web
- https://merginmaps.com/docs/tutorials/working-collaboratively/#share-a-project-with-a-specific-mergin-maps-user

and possibly more. We should change these to the new `merginmaps.com` website.

### PR DESCRIPTION
**Pull request by alex-cit**
fix #173 replaced main platform name in text and also in **vuepress components**. Please check if I did that correctly.
The component <MainDomainNameLink /> refers to merginmaps.com, although there was an instance (https://merginmaps.com/docs/setup/sign-up-to-mergin-maps/#forgotten-password) which would need "app.merginmaps.com". In other cases, it seems to me to better link the main page (merginmaps.com) rather than the login page (app.merginmaps.com).
There are still 2 references to cloudmergin.com:
- https://merginmaps.com/docs/layer/plugin-variables/ - I tried this in QGIS and the variable '@mergin_url' still refers to https://public.cloudmergin.com in QGIS, so I did not change it in this docs yet
- https://merginmaps.com/docs/dev/integration/ is maybe a similar case: " --url       Mergin url      (defaults to public.cloudmergin.com)". This is still valid?

### CODE REVIEW

**Comment by PeterPetrik** [comment]
public.cloudmergin.com is proxied to app.merginmaps.com so you can have the new URL in docs without problem

I would change both of these, we will release it soon 
-  mergin_url: see https://github.com/lutraconsulting/qgis-mergin-plugin/issues/414  
- re integration: see https://github.com/lutraconsulting/mergin-py-client/pull/137

File: src/index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ home: false
 The ecosystem consist of various components:
  - [QGIS](https://qgis.org/) > Powerful GIS Desktop application 
  - [QGIS Mergin Plugin](https://plugins.qgis.org/plugins/Mergin/) > <QGISPluginName />
- - [Mergin Cloud](https://public.cloudmergin.com) > SaaS Cloud Service (available also as Mergin CE)
+ - [Mergin Cloud](https://merginmaps.com) > SaaS Cloud Service (available also as Mergin CE)
  - [Mergin Maps Input](https://inputapp.io) > the <MobileAppName /> for iOS and Android
```
Comment (PeterPetrik) [inline src/index.md:19]: https://inputapp.io is also fully redirected to https://merginmaps.com now, so best to get rid of it to everywhere

File: src/misc/contribute.md [hunk 2]
```diff
@@ -53,5 +53,5 @@ or for other subprojects
 ## Donate or subscribe
 
  - Donate
- - [Subscribe](https://public.cloudmergin.com) to the Mergin Cloud
+ - [Subscribe](https://merginmaps.com/) to the Mergin Cloud
```
Comment (PeterPetrik) [inline src/misc/contribute.md:56]: you can use component here?

File: src/setup/sign-up-to-mergin-maps/index.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ check your SPAM folder if the confirmation email does not appear in your inbox a
 
 ## Forgotten password
 
-If you forget your password, you can reset it through the <MainDomainNameLink id="login/reset" desc="Mergin website" />.
+If you forget your password, you can reset it through the [Mergin website](https://app.merginmaps.com/login/reset).
```
Comment (PeterPetrik) [inline src/setup/sign-up-to-mergin-maps/index.md:64]: you can use component here?

Comment (PeterPetrik) [reply src/setup/sign-up-to-mergin-maps/index.md:64]: or you can create new component for "app" (AppDomainNameLink, etc)

**Comment by alex-cit** [comment]
added new component AppDomainLinkName
- added components to https://merginmaps.com/docs/misc/write-docs/
- replaced inputapp.io links
- used components instead of plain text when possible (e.g. Input)

**Review by PeterPetrik — APPROVED** [review]

**Pull request merged**